discovery_engine_core/core/src/storage/migrations/20220801152246_source_preference.sql

```diff
@@ -0,0 +1,19 @@
+--  Copyright 2022 Xayn AG
+--
+--  This program is free software: you can redistribute it and/or modify
+--  it under the terms of the GNU Affero General Public License as
+--  published by the Free Software Foundation, version 3.
+--
+--  This program is distributed in the hope that it will be useful,
+--  but WITHOUT ANY WARRANTY; without even the implied warranty of
+--  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--  GNU Affero General Public License for more details.
+--
+--  You should have received a copy of the GNU Affero General Public License
+--  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+CREATE TABLE IF NOT EXISTS SourcePreference (
+    source TEXT NOT NULL PRIMARY KEY,
+    -- 1 = preferred, 2 = excluded
```

Comment:
Do we want to keep the name `preferred` or would it be better to use `trusted`?

Reply:
I'd say `trusted` as that's how we have been calling it

Reply:
i have changed it to `trusted`